src/fmt.rs

```diff
@@ -11,10 +11,27 @@
 
 //! Adapters for alternative string formats.
 
-use crate::{
-    std::{borrow::Borrow, fmt, ptr, str},
-    Uuid, Variant,
-};
+use alloc::vec::Vec;
+use crate::{Bytes, std::{borrow::Borrow, fmt, ptr, str}, Uuid, Variant};
+
+impl From<Uuid> for Vec<u8>{
+    fn from(value: Uuid) -> Self {
+        value.as_bytes().to_vec()
+    }
+}
+
+impl From<Vec<u8>> for Uuid{
```

Comment:
I don't think we should add this side of the conversion. Converting slices into `Uuid`s is fallible, but we can't express that through this trait.